src/phoenix/settings.py

```diff
@@ -1,12 +1,20 @@
 from dataclasses import dataclass, field
 
+from phoenix.config import LoggingMode
+
 
 @dataclass
 class _Settings:
     """Settings for Phoenix, lazily initialized."""
 
     # By default, don't log migrations
     log_migrations: bool = field(default=False)
+    # By default, Phoenix does not configure its loggers and acts as a library
+    logging_mode: LoggingMode = field(default=LoggingMode.AS_LIBRARY)
+    # By default, log level is INFO
+    logging_level: int = field(default=10)
+    # By default, log level is WARNING
+    db_logging_level: int = field(default=30)
```

Comment:
```suggestion
    logging_level: int = field(default=logging.INFO)
    # By default, log level is WARNING
    db_logging_level: int = field(default=logging.WARNING)
```

I think INFO is 20, DEBUG is 10. Can use these constants from `logging` for readability.

Reply:
Good catch, what was like that while I was testing